autopilot/accounts.go

```diff
@@ -172,7 +172,7 @@ func (a *accounts) refillWorkerAccounts(w Worker) {
 				// handle registering alert.
 				alertID := types.HashBytes(append(alertAccountRefillID[:], accountID[:]...))
 				if err != nil && inSet {
-					a.ap.alerts.Register(alerts.Alert{
+					rerr := a.ap.alerts.RegisterAlert(ctx, alerts.Alert{
```

Comment:
Now that we've centralised alerts, should we make the "origin" a required property? I know we can pass it in the Data but I think it makes sense to make it required so the UI knows whether the alert is a worker alert or if it originated from the autopilot?